### ISSUE TITLE
Steel Series QCK: RGB Mouse Pad and Rival 700 Mouse support

### ISSUE THREAD

**Issue by Wontell** [antonpup/Aurora#992]
Hi,
Can you please add SteelSeries QCK mouse Pad support, it has 12 RGB zones and can be very comfortable for setting up different things like health, money and all that (right now it only does it in SteelSeries supported games, which are less then 5 of them), things that can "snake up" around the mouse pad. 
Also please support SteelSeries Rival 700 mouse, it has 2 RGB zones and OLED that can display static images or gifs.

BTW, Corsair and Razor also has Multi zone mouse pads, so its not just steel series.

Best Regards

**Comment by simon-wh** [comment]
Steelseries QCK is a duplicate of #841. For the Rival 700 you should be able to light the two main rgb zones through Aurora currently, with the screen I'm not sure what to do about it. I'll have a look at it when I start implemented more devices.

**Comment by Wontell** [comment]
Here is an idea what to do with OLED panel without making it complicated: The OLED panel wont support any games, but be used to PC stats. It has display wide enough for 2 stats at once, so say CPU/GPU temp, after X seconds: CPU/GPU load, after X seconds CPU/GPU memory utilization and so on.

**Comment by simon-wh** [comment]
@Wontell I think I could make a layer which would allow you to use any defined variables (e.g. game information or pc stats) to display text on it. However, as this is a feature that as far as I can tell doesn't have a massive demand, it won't be a high priority for me right now.

**Comment by alessio-cpt** [comment]
I understand this is low priority, but I am also interested in Rival 700 support. Mostly about lights actually, but support for the OLED would also be nice since it is almost useless right now. Something as simple as loading a custom gif for each game or showing basic info would suffice.

@simon-wh you said I should be able to control lights, but the rival 700 does not appear in the device & wrappers list, only the 300 does. Should I use that one?

**Comment by simon-wh** [comment]
@alessio-cpt The rival 700 doesn't work with Aurora currently, @SnakePin was working on trying to get it working last I heard

**Comment by alessio-cpt** [comment]
Ok, thanks, I'll keep an eye on it. If I can give some feedback @SnakePin, as I said before lights should be a priority. Most people around don't seem to care about the OLED, it's just a nice-to-have.

**Comment by Wontell** [comment]
Hi,
The RGB works fine on Rival 700, you must have SteelSeries installed and Aurora will install a plugin that you must Enabled in  Steel series tools.
Then you can choose the rival 300 mouse from the option, that mouse option is just graphical representation, you can choose even Logitech, but it must match the RGB location, so Rival 300 has RGB in both places 700 has.
You can easily test it, by going to Auroa/Desktop settings, choosing any settings, going to Affected keys window and then clicking on the mouse logo and scroll key RGB location, it will add both of these to chose setting and light them up, I just tested and it works.
Basically if you have Steel series devices [I also have RGB mouse mat], the only way Aurora works with them is trough SteelSeries plugin in SteelSeries/Engine apps.
My only issue with Aurora supports so few games :(

**Comment by alessio-cpt** [comment]
Got it. Thanks for the instructions!

**Comment by Gurjot95** [comment]
Should be working in latest build.

**Comment by nmarrufo** [comment]
I am not able to get the Rival 700 working with the Prism mousepad. As of right now Aurora is set to razer/corsair mousepad to get the mouse working but when i set it to the steelseries 300 option it strobes. And nothing I do can get the Prism to recognize. Any ideas?
